### ISSUE TITLE
Crash on Samsung Galaxy A02s

### ISSUE THREAD

**Issue by saberraz** [lutraconsulting/input#1705]
In the new release (1.0.2), there are new crashes reported for Samsung Galaxy A02s (32 bit). The stacktrace does not appear very helpful:
 
```
*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
pid: 0, tid: 0 >>> uk.co.lutraconsulting <<<

backtrace:
  #00  pc 00000000001e0aec  /data/app/~~UoJrrJ2hsZSREev6LyVesw==/uk.co.lutraconsulting-hdHCAZPctcRBFqmbBviz8Q==/lib/arm/libQt5Core_armeabi-v7a.so
```

EDIT:

The same crash appears on 64bit versions

**Comment by tomasMizera** [comment]
I found out that this one is the same as #1717 

Logs:

```
11-02 09:04:18.709  4240  4974 W Input   : qrc:/form/FeatureForm.qml:490:9: QML Loader: Binding loop detected for property "value"
11-02 09:04:18.714  4240  4974 W Input   : qrc:/form/FeatureForm.qml:490:9: QML Loader: Binding loop detected for property "value"
11-02 09:04:18.719  4240  4974 W Input   : qrc:/form/FeatureForm.qml:490:9: QML Loader: Binding loop detected for property "value"
11-02 09:04:18.724  4240  4974 W Input   : qrc:/form/FeatureForm.qml:490:9: QML Loader: Binding loop detected for property "value"
11-02 09:04:18.730  4240  4974 W Input   : qrc:/form/FeatureForm.qml:490:9: QML Loader: Binding loop detected for property "value"
11-02 09:04:18.735  4240  4974 W Input   : qrc:/form/FeatureForm.qml:490:9: QML Loader: Binding loop detected for property "value"
11-02 09:04:18.740  4240  4974 W Input   : qrc:/form/FeatureForm.qml:490:9: QML Loader: Binding loop detected for property "value"
11-02 09:04:18.746  4240  4974 W Input   : qrc:/form/FeatureForm.qml:490:9: QML Loader: Binding loop detected for property "value"
11-02 09:04:18.751  4240  4974 W Input   : qrc:/form/FeatureForm.qml:490:9: QML Loader: Binding loop detected for property "value"
11-02 09:04:18.756  4240  4974 W Input   : qrc:/form/FeatureForm.qml:490:9: QML Loader: Binding loop detected for property "value"
11-02 09:04:18.757  4240  4974 F libc    : Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x715b1a5ba0 in tid 4974 (qtMainLoopThrea), pid 4240 (lutraconsulting)
11-02 09:04:18.847  5059  5059 I crash_dump64: obtaining output fd from tombstoned, type: kDebuggerdTombstone
11-02 09:04:18.847   671   671 I tombstoned: received crash request for pid 4974
11-02 09:04:18.848  5059  5059 I crash_dump64: performing dump of process 4240 (target tid = 4974)
11-02 09:04:18.859  5059  5059 F DEBUG   : *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
11-02 09:04:18.859  5059  5059 F DEBUG   : Build fingerprint: 'samsung/a32eea/a32:11/RP1A.200720.012/A325FXXU1AUH1:user/release-keys'
11-02 09:04:18.859  5059  5059 F DEBUG   : Revision: '4'
11-02 09:04:18.859  5059  5059 F DEBUG   : ABI: 'arm64'
11-02 09:04:18.860  5059  5059 F DEBUG   : Timestamp: 2021-11-02 09:04:18+0100
11-02 09:04:18.860  5059  5059 F DEBUG   : pid: 4240, tid: 4974, name: qtMainLoopThrea  >>> uk.co.lutraconsulting <<<
11-02 09:04:18.860  5059  5059 F DEBUG   : uid: 10342
11-02 09:04:18.860  5059  5059 F DEBUG   : signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x715b1a5ba0
11-02 09:04:18.860  5059  5059 F DEBUG   : Cause: stack pointer is in a non-existent map; likely due to stack overflow.
11-02 09:04:18.860  5059  5059 F DEBUG   :     x0  000000715b1adc08  x1  00000071f879ec48  x2  0000000000000013  x3  0000000000000000
11-02 09:04:18.860  5059  5059 F DEBUG   :     x4  0000000000000000  x5  00000070dd9f00e0  x6  00000070dd9f0080  x7  0000000000000000
11-02 09:04:18.860  5059  5059 F DEBUG   :     x8  825a42ca0ef5c991  x9  000000715b1a5bc8  x10 000000715b1a5ba8  x11 000000715b1adbc8
11-02 09:04:18.860  5059  5059 F DEBUG   :     x12 0000000000000002  x13 0000000000000000  x14 0000000000000000  x15 000000715b1b2e10
11-02 09:04:18.860  5059  5059 F DEBUG   :     x16 00000070f8e7a8a8  x17 00000071fb3f38a0  x18 0000007105860000  x19 000000715b1a5ba0
11-02 09:04:18.860  5059  5059 F DEBUG   :     x20 000000715b2ae000  x21 00000071676aed20  x22 000000715b2ae000  x23 00000071675d1120
11-02 09:04:18.860  5059  5059 F DEBUG   :     x24 0000000000000000  x25 0000000000000001  x26 00000070dd9f0120  x27 0000000000000001
11-02 09:04:18.860  5059  5059 F DEBUG   :     x28 00000070f34271c0  x29 000000715b1adbf0
11-02 09:04:18.860  5059  5059 F DEBUG   :     lr  00000070f8bbcf8c  sp  000000715b1a5ba0  pc  00000070f8bbd030  pst 0000000060000000
11-02 09:04:18.861  5059  5059 F DEBUG   : backtrace:
11-02 09:04:18.861  5059  5059 F DEBUG   :       #00 pc 00000000002a9030  /data/app/~~kwMmcsIQofytBhBMZ4sP4w==/uk.co.lutraconsulting-UaQqz6J9kuVGldYAUYFtFQ==/lib/arm64/libQt5Core_arm64-v8a.so (BuildId: 08753fe183074e12059b5714a490ca7a5b1ae754)
```

**Comment by saberraz** [comment]
This crash is still happening and is the top one in the list.

**Comment by tomasMizera** [comment]
Fix should be resolved in released 1.2.0 version
